### PR TITLE
New padding

### PR DESCRIPTION
**Pull request by alejoe91**
@samuelgarcia

### CODE REVIEW

File: MEArec/generators/recordinggenerator.py [hunk 1]
```diff
@@ -145,7 +143,7 @@ def __del__(self):
         self.spike_traces = None
 
         if not self._is_tmp_folder_local:
-            if self._verbose:
+            if self._verbose >= 1:
```
Comment (samuelgarcia) [inline MEArec/generators/recordinggenerator.py]: if self._verbose >= 1: is tha same as if self._verbose:

File: MEArec/generators/recordinggenerator.py [hunk 2]
```diff
@@ -193,6 +191,18 @@ def generate_recordings(self, tmp_mode=None, tmp_folder=None, verbose=None, n_jo
...
+        elif isinstance(verbose, bool):
+            if verbose:
+                verbose_1 = True
+                verbose_2 = False
+            else:
+                verbose_1 = False
+                verbose_2 = False
+        else:
+            verbose_1 = False
+            verbose_2 = False
```
Comment (samuelgarcia) [inline MEArec/generators/recordinggenerator.py]: It was simple and you made it complicated :)

File: MEArec/generators/recordinggenerator.py [hunk 3]
```diff
@@ -940,8 +950,14 @@ def generate_recordings(self, tmp_mode=None, tmp_folder=None, verbose=None, n_jo
 
             chunk_indexes = make_chunk_indexes(duration, chunk_duration, fs)
 
+            # compute pad samples as 3 times the low-cutoff period
+            if cutoff.size == 1:
+                pad_samples_filt = 3 * int((1. / cutoff * fs).magnitude)
+            elif cutoff.size == 2:
+                pad_samples_filt = 3 * int((1. / cutoff[0] * fs).magnitude)
+
```
Comment (samuelgarcia) [inline MEArec/generators/recordinggenerator.py]: Cool

File: MEArec/generators/recordinggenerator.py (hunk 2 above)
Comment (alejoe91) [reply MEArec/generators/recordinggenerator.py]: I also need verbosity in the functions :)

**Pull request merged**